### ISSUE TITLE
Cocktail.Schedule.to_string/1 displays incorrectly when only one day is supplied

### ISSUE THREAD

**Issue by ClementineOldfield** [peek-travel/cocktail#137]
```
"Etc/Utc"
|> DateTime.now!() 
|> Cocktail.Schedule.new() 
|> Cocktail.Schedule.add_recurrence_rule(:weekly, days: [:monday]) 
|> Cocktail.Schedule.to_string()
```

Expected `Weekly on Mondays`
Actual `Weekly on  and Mondays`

**Comment by arialblack14** [comment]
@ClementineOldfield seems that the hex package has not been updated. 
This issue has already been solved (check #110 ).
Use the repo in your mix.exs if you can.

**Comment by doughsay** [comment]
0.9.0 has been published and includes this fix! :tada:
